src/Rings/mpoly-localizations.jl

```diff
@@ -337,19 +337,29 @@ end
 # MPolyComplementOfPrimeIdeal. In order to treat them in 
 # parallel, we introduce the analogous method here. 
 function prime_ideal(S::MPolyComplementOfKPointIdeal) 
+  kk = coefficient_ring(ambient_ring(S))
+  # TODO: Test whether kk is an integral domain
   if !isdefined(S, :m)
     R = ambient_ring(S)
     x = gens(R)
     n = ngens(R)
     m = ideal(R, [x[i]-a for (i, a) in enumerate(point_coordinates(S))])
     set_attribute!(m, :is_prime=>true)
-    set_attribute!(m, :is_maximal=>true)
-    set_attribute!(m, :dim=>0)
+    set_attribute!(m, :is_maximal=>iszero(dim(kk)))
```

Comment:
I am in doubt that this will work for kk = ZZ/<4>.

Reply:
Yes. But nothing really does, so I think, it cannot be a real issue here. It wouldn't be mathematically incorrect here, would it?

Reply:
This attribute is_maximal is mathematically incorrect here, as a maximal ideal in this example would be m+<2>, which strictly contains m.